lib/ansible/plugins/lookup/aws_ssm.py

```diff
@@ -100,6 +100,24 @@
   debug: msg='key contains {{item.Name}} with value {{item.Value}} '
   loop: '{{ query("aws_ssm", "/TEST/test-list", region="ap-southeast-2", bypath=true) }}'
 
+# Example using STS temporary credentials
+
+---
+  - name: STS AssumeRole
+    sts_assume_role:
+      role_arn: 'arn:aws:iam::<acc_id>:role/<role_name>'
+      role_session_name: "{{ aws_sts_session_name | default('ansible-session') }}"
+      region: '<region_name>'
```

Comment:
In case people copy and paste this, either make '<region_name>' a variable or replace the string with a valid one. Below too. But I think creating temporary credentials for plugins/modules should be documented somewhere more central since this isn't specific to this plugin.

Also, since this plugin uses boto3 a nicer way to do this is use a profile in your config file (usually found at ~/.aws/config) that has a source profile and the role to assume. For example:
```
# In ~/.aws/credentials:
[development]
aws_access_key_id=foo
aws_access_key_id=bar

# In ~/.aws/config
[profile crossaccount]
role_arn=arn:aws:iam:...
source_profile=development
```
and then use "{{ lookup('aws_ssm', 'my-parameter', aws_profile='crossaccount'}}" to assume the role instead of needing this first setup task.